snippets/csharp/language-reference/operators/ConditionalExamples.cs

```diff
@@ -1,45 +1,67 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace operators
 {
     public class ConditionalExamples
     {
         public static void Examples()
         {
+            ConditionalRefExpressions();
             ConditionalValueExpressions();
+            ComparisonWithIf();
         }
 
-        static void ConditionalRefExpressions()
+        private static void ConditionalRefExpressions()
         {
             // <SnippetConditionalRef>
             var smallArray = new int[] { 1, 2, 3, 4, 5 };
             var largeArray = new int[] { 10, 20, 30, 40, 50 };
 
-            int index= new Random().Next(0, 9);
+            int index = 7;
             ref int refValue = ref ((index < 5) ? ref smallArray[index] : ref largeArray[index - 5]);
-            Console.WriteLine(refValue);
+            refValue = 0;
+
+            index = 2;
+            ((index < 5) ? ref smallArray[index] : ref largeArray[index - 5]) = 100;
```

Comment:
To show that `ref` in front of the conditional ref expression is not necessary.
Also, I've made the examples non-random, in order to present the output in comments. Examples with `ref` doesn't run in the interactive mode.